krkn/scenario_plugins/node_actions/node_actions_scenario_plugin.py

```diff
@@ -153,7 +153,114 @@ def inject_node_scenario(
             nodes = common_node_functions.get_node(
                 label_selector, instance_kill_count, kubecli
             )
-        
+            
+        # Get scenario-specific parameters
```

Comment:
This section looks pretty identical to what is in [run_node](https://github.com/krkn-chaos/krkn/blob/a4b542f9ce993d09569a938b1ed92c8507003981/krkn/scenario_plugins/node_actions/node_actions_scenario_plugin.py#L308), can you add the new scenario as an if statement into run_node instead 

Adding this also will execute the scenario twice, once in what you've added and once [here](https://github.com/krkn-chaos/krkn/blob/a4b542f9ce993d09569a938b1ed92c8507003981/krkn/scenario_plugins/node_actions/node_actions_scenario_plugin.py#L264-L269)